src/test/rpc/NoRippleCheck_test.cpp

```diff
@@ -258,6 +258,9 @@ class NoRippleCheckLimits_test : public beast::unit_test::suite
         {
             if (! admin)
             {
+                static int lastBalance;
```

Comment:
Would be great to add a paragraph comment about why and how `lastBalance` is being used.  Perhaps...

1. Describe the problem you were seeing.
2. Describe how this implementation addresses the problem.

Otherwise a maintainer won't know why they should preserve this machinery.  That comment will also make it easier for me to review the code, since I'm not really sure what the change is trying to accomplish.

Thanks.

Reply:
I've added some comments - I hope that captures what you were looking for, but let me know if something's still missing.